shared/src/main/scala/coop/rchain/store/KeyValueTypedStoreSyntax.scala

```diff
@@ -14,8 +14,16 @@ final class KeyValueTypedStoreOps[F[_], K, V](
     // KeyValueTypedStore extensions / syntax
     private val store: KeyValueTypedStore[F, K, V]
 ) extends AnyVal {
+  def errKVStoreExpectValue = "Error when reading from KeyValueStore: value not found."
+
   def get(key: K)(implicit f: Functor[F]): F[Option[V]] = store.get(Seq(key)).map(_.head)
 
+  def getUnsafe(keys: Seq[K])(implicit f: Sync[F]): F[List[V]] =
```

Comment:
I'm little sceptical about this unsafe calls without any indication of used keys.

Maybe just simple `"$key"` would be better then nothing.

Reply:
Maybe force is to have implicit `Show[K]`?